components/evergreen.ts

```diff
@@ -51,5 +52,11 @@ export async function getCpd(
         return defaultCPD
     }
 
+    //NOTE: Override the EvergreenLevel with the latest Mastery Level
+    if (getFlag("gameplayUnlockAllFreelancerMasteries")) {
+        //TODO: Get rid of hardcoded values
```

Comment:
I thought about that, but on the Discord we also had talk where it's okay progression happens, as long as people are not locked out of unlocks. Honestly, this was the easy way to go since already had code at that spot initially. In testing, flipping this flag will kind of mess-up Safehouse, since some things will be undone, and some won't.

Reply:
Well at some point in the future, the inventory will be managed by whatever they unlocked through challenges or mastery. So having all this behind a generic "unlock all" flag/button would be ideal even if that's not a feature now.

About flipping it, the idea behind an "unlock all" button would be that the only way to undo it is to reset your profile.